include/eve/arch/cpu/logical_wide.hpp

```diff
@@ -78,6 +78,12 @@ namespace eve
     //! Type representing the numerical value associated to the mask
     using mask_type = wide<Type, Cardinal>;
 
+    //! Type representing a logical wide of the same type but with a cardinal twice the size
+    using combined_type = logical<wide<Type, typename Cardinal::combined_type>>;
+
+    //! Type representing a logical wide of the same type but with a cardinal half the size
+    using split_type    = logical<typename wide<Type, Cardinal>::split_type>;
+
```

Comment:
I'm not sure I like these.

Reply:
Then you should probably `requires` on them.
I don't know if you can `requires` on the nested type, if not - put it in a base class helper. (let me know if you need help to make that work).